.github/workflows/test-file-utils-lib.yml

```diff
@@ -15,44 +15,59 @@ on:
       - '!src/file-utils/**/*.md'
     branches:
       - main
+  workflow_dispatch:
+
 jobs:
   test-file-utils:
     name: Run unit, style, and type checks
     runs-on: ubuntu-20.04
+    env:
+        PACKAGE_NAME: file-utils
+        SRC_DIR: src/file-utils
+        LOCK_FILE: src/file-utils/poetry.lock
     steps:
       - name: Checkout code
         uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
 
       - name: Install Python 3.9
         uses: actions/setup-python@3105fb18c05ddd93efea5f9e0bef7a03a6e9e7df
         with:
-          python-version: '3.9.x'
+          python-version: '3.9.4'
```

Comment:
Hard-coding to 3.9.4 was really only important for Canvas. So I need to change this back.